.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -50,6 +50,7 @@ def get_input(question, default=""):
                     help="Sets up Submitty for use with the sample courses. This is a Vagrant convenience "
                          "flag and should not be used in production!")
 parser.add_argument('--worker', action='store_true', default=False, help='Configure Submitty with autograding only')
+parser.add_argument('--worker-pair', default=False, help='Configure Submitty alongside a worker VM')
```

Comment:
Make the help string more explicit that this should only be run when using vagrant.